types/c-struct/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-class": false
```

Comment:
If a lint rule absolutely must be suppressed, please use a line-level suppression to make it easier to review.